addons/survey/static/src/js/utils.js

```diff
@@ -0,0 +1,38 @@
+async function animate(el, keyFrame, duration) {
```

Comment:
move this file one level up as well, we want to remove the js folder at some point